python/perf-kernels/flash-attention.py

```diff
@@ -824,12 +924,12 @@ def backward(ctx, do, _):
                           (4, 48, 4096, 128),
                           (4, 16, 8192, 64),
                           (4, 16, 8080, 64),
-                          (1, 48, 16384, 64),
+                          (1, 16, 16384, 64),
                           (4, 48, 127, 64),
                           ])
 @pytest.mark.parametrize('causal', [False])
 @pytest.mark.parametrize('use_bias', [True, False])
-@pytest.mark.parametrize('bias_type', ["vector", "matrix"])
+@pytest.mark.parametrize('bias_type', ["vector"])
```

Comment:
Just a reminder "matrix" `bias_type` is not tested here.
(I'll add a `# FIXME` or `# TODO` Here)

Reply:
I removed this to reduce number of configs being tested and forgot to add it back. Fixed.